### ISSUE TITLE
error-strings does not allow multiple functions for the same package

### ISSUE THREAD

**Issue by leungster** [mgechev/revive#1243]
**Describe the bug**
The `error-strings` rule uses the package name as the key to the [errorFunctions](https://github.com/mgechev/revive/blob/master/rule/error_strings.go#L46) map. If the arguments list has multiple functions under the same package like `a.Wrap` and `a.New`, the last function is linted against.

**To Reproduce**
Configure `error-strings` with error functions in the same package. e.g. `errors.New` will overwrite the default map.

**Expected behavior**
Allow multiple functions in the same package to be linted against.

**Comment by alexandear** [comment]
Hi @leungster 

Thank you for reporting this issue. To help us investigate further, could you please provide minimal steps to reproduce the problem? Specifically, we need a sample file with the relevant code and the actual/expected output.

For example:


Code:

```go
// main.go
package main

import (
    "errors"
    "fmt"
)

func main() {
    err := errors.New("an error occurred")
    fmt.Println(err)
}
```

Configuration:

```toml
[rule.error-strings]
    arguments = ["errors.New"]
```

Actual:
```sh
$ revive main.go
// output showing the issue
```

Expected:
```sh
$ revive main.go
// expected output showing correct linting
```

**Comment by leungster** [comment]
Test data:
```golang
// main.go
package main

import (
    "errors"
    "fmt"
)

func main() {
    err := errors.New("An error occurred!")
    fmt.Println(err)
}
```

Configuration
```toml
[rule.error-strings]
    arguments = ["errors.New", "errors.Wrap"]
```

Actual:
```
$ revive main.go
// no errors
```

Expected:
```
$ revive main.go
main.go:10:20: error strings should not be capitalized or end with punctuation or a newline

```

**Comment by alexandear** [comment]
@leungster, thank you. I was able to reproduce and fix the bug.
